### PR TITLE
store/tikv: set different timeout for tikv requests.

### LINKED ISSUE #1694: Increase gc_worker's requests' network timeout.
`CmdGC` usually takes time much longer than regular commands, use the same timeout could lead to frequently timeout and retry.

### PR DESCRIPTION
**Pull request by disksing**
@shenli @ngaut

Fix #1694

### CODE REVIEW

File: store/tikv/client.go [hunk 1]
```diff
@@ -31,14 +32,18 @@ type Client interface {
...
 
 const (
-	maxConnecion = 150
-	netTimeout   = 20 // seconds
+	maxConnecion        = 150
+	dialTimeout         = time.Duration(5) * time.Second
+	writeTimeout        = time.Duration(10) * time.Second
+	readTimeoutShort    = time.Duration(20) * time.Second  // For requests that read/write several key-values.
+	readTimeoutLong     = time.Duration(60) * time.Second  // For requests that may need scan region.
+	readTimeoutVeryLong = time.Duration(150) * time.Second // For requests that may need scan region multiple times.
```
Comment (zimulala) [inline store/tikv/client.go]: Remove `time.Duration`.

Comment (coocood) [inline store/tikv/client.go]: How about `medium` and `long`?

**Comment by coocood** [comment]
LGTM

**Comment by zimulala** [comment]
LGTM

**Pull request merged**